spafe/utils/preprocessing.py

```diff
@@ -132,6 +141,9 @@ def windowing(frames, frame_len, win_type="hamming"):
                                    https://superkogito.github.io/blog/2020/03/13/spectral_leakage_and_windowing.html
 
     """
+    # WARNING : I would argue that defaulting to hamming is good,
+    # but defaulting _silently_ is bad (any spelling mistake made would result
+    # in a silent usage of hamming)
```

Comment:
What do you recommend to do here instead? I am a bit confused by your point.

Reply:
Two possible behaviors IMHO:
1. since there's already a default argument in the function, remove the `get` and straight up use a regular `window_fn[window_type]` (getitem) syntax. Any badly written window type will raise a `KeyError` and it's a very quick fix for the user.
2. keep the same behavior and fallback to hamming, but issue a `warnings.warn` to notify the user that this window type is being used.

I'd rather go for behavior 1, but it's your call on this one.

Reply:
Option 1 seems better so I went with that. Thank you :))